exercises/firmware_auditing/README.md

```diff
@@ -0,0 +1,318 @@
+<!--
+Copyright lowRISC Contributors.
+SPDX-License-Identifier: Apache-2.0
+-->
+# Firmware Auditing Exercise
+
+First, make sure to go to the [building the exercises][] section to see how the exercises are built.
+
+[building the exercises]: ../README.md#building-the-exercises
+
+Next, go to the [auditing firmware][] documentation section to get a brief introduction to `cheriot-audit` and the Rego policy language.
+Part 3 of the [hardware access control] exercise also shows a simple example of using CHERIoT Audit to ensure that compartments do not access MMIO capabilities they should not be accessing.
```

Comment:
```suggestion
You may also be interested in doing the [hardware access control][] exercise in uses CHERIoT Audit in it's part 3 to ensure that compartments do not access MMIO capabilities they should not be accessing.
```

Reply:
Moved this to the last section instead as suggested later, but I've still tried to make it more of a suggestion there and clearly not a requirement.